doc/Type/Supply.pod6

```diff
@@ -371,10 +371,15 @@ C<&mapper> and emits it to the new supply.
     method batch(Supply:D: :$elems, :$seconds --> Supply:D)
 
 Creates a new supply that batches the values of the given supply by either
-the number of elements in the batch (using C<:elems>) or the maximum number of
-seconds (using the C<:seconds>) or both.  Any remaining values are emitted in
+the number of elements in the batch (using C<:elems>) or a duration
+(using C<:seconds>) or both.  Any remaining values are emitted in
 a final batch when the supply is done.
 
+B<Note>: Since Rakudo version 2020.12, the C<:seconds> parameter has
+a millisecond granularity: for example a 1 millisecond duration could
+be specified as C<:seconds(0.001)>. Before Rakudo version 2020.12,
+the C<:seconds> parameter has a second granularity.
```

Comment:
I would say that it "had" the granularity.

Reply:
Sure. Done !